modules/outbound-command-event-handler/src/api-server/models/health.ts

```diff
@@ -25,4 +25,5 @@
 export interface Health {
     status: 'OK' | 'ERROR';
     errors?: string[];
+    bulkTransactionRepoConnected?: boolean;
```

Comment:
What about Kafka?

Reply:
I had checked @mojaloop/platform-shared-lib-nodejs-kafka-client-lib and couldn't find anything that was a clear cut health check.

I had planned to creating a story for that or try and figure out a faux health check.